src/GafferOSL/OSLShader.cpp

```diff
@@ -307,6 +336,49 @@ Plug *loadNumericParameter( const OSLQuery::Parameter *parameter, const Interned
 	return plug.get();
 }
 
+template<typename PlugType>
+Plug *loadNumericArrayParameter( const OSLQuery::Parameter *parameter, const InternedString &name, Gaffer::Plug *parent, const CompoundData *metadata )
+{
+	typedef typename PlugType::ValueType DataType;
+	typedef typename DataType::ValueType ValueType;
+
+	ValueType defaultValue;
+	if( parameter->idefault.size() )
+	{
+		defaultValue.resize( parameter->idefault.size() );
+		for( int i = 0; i < parameter->idefault.size(); i++ )
```

Comment:
Should be `size_t` rather than `int` - this is the cause of the compilation failures on Travis. Maybe we can just use `std::copy()` rather than these explicit loops anyway? Or maybe even vector assignment?